Bolts/build.gradle

```diff
@@ -26,6 +26,8 @@ android {
 
 dependencies {
     testCompile 'junit:junit:4.12'
+
+    androidTestCompile 'com.android.support:support-v4:22.2.1'
```

Comment:
The latest version is `com.android.support:support-v4:23.0.1`, any reason we do not use that?